package.json

```diff
@@ -56,7 +56,7 @@
     "xhr-mock": "^2.5.1"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "^1.3.1",
+    "@eppo/js-client-sdk-common": "^1.4.1",
```

Comment:
I might bump this a higher version if this gets merged/published first: https://github.com/Eppo-exp/js-client-sdk-common/pull/13